test/e2e_node/node_shutdown_linux_test.go

```diff
@@ -24,22 +24,22 @@ import (
 	"strconv"
 	"time"
 
-	"k8s.io/apimachinery/pkg/fields"
-
 	"github.com/onsi/ginkgo"
 	"github.com/onsi/gomega"
-	"k8s.io/kubernetes/pkg/apis/scheduling"
-	"k8s.io/kubernetes/pkg/features"
-	"k8s.io/kubernetes/test/e2e/framework"
 
 	v1 "k8s.io/api/core/v1"
+	schedulingv1 "k8s.io/api/scheduling/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/kubernetes/pkg/apis/scheduling"
+	"k8s.io/kubernetes/pkg/features"
 	kubeletconfig "k8s.io/kubernetes/pkg/kubelet/apis/config"
 	kubelettypes "k8s.io/kubernetes/pkg/kubelet/types"
+	"k8s.io/kubernetes/test/e2e/framework"
 	testutils "k8s.io/kubernetes/test/utils"
 )
 
-var _ = SIGDescribe("GracefulNodeShutdown [Serial] [NodeAlphaFeature:GracefulNodeShutdown]", func() {
+var _ = SIGDescribe("GracefulNodeShutdown [Serial] [NodeBetaFeature:GracefulNodeShutdown] [NodeAlphaFeature:PodPriorityBasedGracefulShutdown]", func() {
```

Comment:
```suggestion
var _ = SIGDescribe("GracefulNodeShutdown [Serial] [NodeFeature:GracefulNodeShutdown] [NodeAlphaFeature:PodPriorityBasedGracefulShutdown]", func() {
```

This should just become NodeFeature, we don't use NodeBetaFeature in the test suites.

What job is running these e2es? @bobbypage ?

Reply:
This is running under job here: 

https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-node/node-kubelet.yaml#L411

We'll need to update the test-infra config since now we're only running (`NodeAlphaFeature`)

```
          - --test_args=--nodes=1 --focus="\[NodeAlphaFeature:GracefulNodeShutdown\]"
```

Reply:
I sent a patch to update this kubernetes/test-infra#22728